kore-rpc-types/kore-rpc-types.cabal

```diff
@@ -0,0 +1,127 @@
+cabal-version: 2.2
+name:           kore-rpc-types
+version:        0.60.0.0
+description:    Please see the [README](README.md) file.
+category:       Language
+homepage:       https://github.com/runtimeverification/haskell-backend#readme
+bug-reports:    https://github.com/runtimeverification/haskell-backend/issues
+author:         Runtime Verification Inc
+maintainer:     ana.pantilie@runtimeverification.com
+copyright:      2018-2021 Runtime Verification Inc
+license:        BSD-3-Clause
+build-type:     Simple
+
+source-repository head
+  type: git
+  location: https://github.com/runtimeverification/haskell-backend
+
+flag threaded
+  description: Use the threaded runtime. Recommended to disable for profiling.
+  manual: True
+  default: True
+
+common haskell
+  -- Common Haskell dialect
+  default-extensions:
+    BangPatterns
+    -- GHC2021 prep
+    BinaryLiterals
+    -- end
+    ConstraintKinds
+    DataKinds
```

Comment:
Should we clean up the cabal file before merging? Especially the extension list with the `GHC2021 prep` notes 
(we might also make a pass over the "main" cabal file and remove stuff that does not belong)

Reply:
good point. i will focus on this list in this PR